apps.json

```diff
@@ -114,5 +114,16 @@
     "icon_url":
       "https://assets.getpebble.com/api/file/PYUmPYRjSKqAtYfXdUv9/convert?cache=true&fit=crop&w=144&h=168",
     "tags": ["next", "bus", "stop"]
+  },
+  {
+    "name": "uWave",
+    "description":
+      "uWave is a student all-in-one mobile app 📆 Student Timetable, ⏱ Finals countdown in Timetable, 🚌 Public Bus Timings + Campus Shuttle Location, 📍 NUS Campus Maps, 💬 NUS Discussions, 🏓 List of all CCAs in NUS, 🧧 Student discounts",
+    "author": "Wave",
+    "url":
+      "https://uwave.sg/download",
+    "icon_url":
+      "https://assets.getpebble.com/api/file/PYUmPYRjSKqAtYfXdUv9/convert?cache=true&fit=crop&w=144&h=168",
```

Comment:
@wave-sg your icon is broken :)

Reply:
How did I totally miss that! I even had my image URL on standby @ https://play-lh.googleusercontent.com/a9mD7W6FDPFmGVllWCcja_3I9ynoHHSITEn_bVhq01X7aHMXC-tv8SKCDxLu04Q6-6U%3Ds360-rw